### PR TITLE
Add version data to admin page

### PR DESCRIPTION
**Pull request by HenryNguyen5**
image

### CODE REVIEW

File: explorer/client/craco.config.js [hunk 1]
```diff
@@ -21,6 +26,12 @@ module.exports = {
         threshold: 0,
         minRatio: 0.8,
       }),
+      new webpack.DefinePlugin({
+        __EXPLORER_CLIENT_VERSION__: JSON.stringify(clientPkg.version),
```
Comment (rupurt) [inline explorer/client/craco.config.js]: Nice 🏀

**Comment by j16r** [comment]
Version is v0.0.1, but our current version is 0.8.0 - is it being reported correctly here?

**Comment by HenryNguyen5** [comment]
> Version is v0.0.1, but our current version is 0.8.0 - is it being reported correctly here?

The version being reported is gathered from `explore/package.json`, we should keep this version in sync with our git tags

**Comment by HenryNguyen5** [comment]
> Could we use a lighter font to match the chainlink gui?

New font: 
image

**Comment by j16r** [comment]
`Backend v0.8.0 | Client v0.0.1`

These should be the same.

**Comment by j16r** [comment]
Ready to accept but waiting on @hellobart

**Pull request merged**